CPAC/seg_preproc/utils.py

```diff
@@ -388,40 +388,27 @@ def hardcoded_antsJointLabelFusion(anatomical_brain, anatomical_brain_mask,
     return multiatlas_Intensity, multiatlas_Labels
 
 
-def pick_tissue_from_labels_file(multiatlas_Labels, csf_label=24,
-                                 left_gm_label=3, left_wm_label=2,
-                                 right_gm_label=42, right_wm_label=41,
-                                 include_ventricles=False):
+def pick_tissue_from_labels_file(multiatlas_Labels, csf_label=[4,14,15,24,43],
+                                 gm_label=[3,42], wm_label=[2,41]):
     """Pick tissue mask from multiatlas labels file
     based off of FreeSurferColorLUT https://surfer.nmr.mgh.harvard.edu/fswiki/FsTutorial/AnatomicalROI/FreeSurferColorLUT
     or user provided label value
 
     Parameters
     ----------
-
     multiatlas_Labels : string (nifti file)
 
-    csf_label : integer
-        label value corresponding to CSF in multiatlas file
-
-    left_gm_label : integer
-        label value corresponding to Left Gray Matter in multiatlas file
-
-    left_wm_label : integer
-        label value corresponding to Left White Matter in multiatlas file
-
-    right_gm_label : integer
-        label value corresponding to Right Gray Matter in multiatlas file
+    csf_label : list
+        a list of integer label values corresponding to CSF in multiatlas file
 
-    right_wm_label : integer
-        label value corresponding to Right White Matter in multiatlas file
+    gm_label : list
+        a list of integer label value corresponding to Left Gray Matter in multiatlas file
 
-    include_ventricles : boolean
-        whether include labels of ventricles in CSF or not
+    wm_label : list
+        a list of integer label value corresponding to Left White Matter in multiatlas file
```

Comment:
Not "Left Gray Matter" and "Left White Matter", just "Gray Matter" and "White Matter", right? It looks like this is taking left and right.

Reply:
Yes! Now "Gray Matter" includes "Left Gray Matter" and "Right Gray Matter". I think it's more condensed and users don't have to distinguish left or right labels.

Reply:
:+1: 

I meant these docstring lines should get rid of the word "Left":

https://github.com/FCP-INDI/C-PAC/blob/e039df0716789adb965d4482cfdfd4c8f7e43519/CPAC/seg_preproc/utils.py#L405

https://github.com/FCP-INDI/C-PAC/blob/e039df0716789adb965d4482cfdfd4c8f7e43519/CPAC/seg_preproc/utils.py#L408

Reply:
Oh great catch, I'll fix it in feature/ccs directly. Thanks!